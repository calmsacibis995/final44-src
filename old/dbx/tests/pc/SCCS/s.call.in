h21388
s 00011/00000/00000
d D 5.1 90/06/25 15:36:44 bostic 1 0
c date and time created 90/06/25 15:36:44 by bostic
e
u
U
t
T
I 1
stop at 33
run
call p1()
call p2(3)
call p3("blah", 3)
call fprintf(&_iob+20, "mainlocal = %d", mainlocal)
print mainlocal
call call
call call()
call p1(3)
call p2("blah")
E 1
