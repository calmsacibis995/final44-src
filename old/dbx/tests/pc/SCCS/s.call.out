h41065
s 00022/00000/00000
d D 5.1 90/06/25 15:36:46 bostic 1 0
c date and time created 90/06/25 15:36:46 by bostic
e
u
U
t
T
I 1
reading symbolic information ...
[1] stop at 33
[1] stopped in startup at line 33
   33       mainlocal := 19;
in p1

p1 returns successfully
in p2(3)

p2 returns successfully
in p3(blah, 3)

p3 returns successfully
mainlocal = 0
fprintf returns successfully
0 
call call
         ^ syntax error
"call" not call-able
too many parameters in call to p1
type mismatch for frommain in call to p2

E 1
